fingertip/main.py

```diff
@@ -62,6 +63,7 @@ def main():
                'or set FINGERTIP_DEBUG=1.'
                if not DEBUG else f'Logfile: {success_log}')
         fingertip.util.log.info(f'Success. {msg}')
+    sys.exit(fingertip.machine.EXIT_CODE)
```

Comment:
just check that `m` has it, use it if it does. am I missing something?

Reply:
Oh right, we don't have m anymore if it was the last one.

Reply:
Yes ;(

Reply:
Since I'm out of good ideas, let's explore some of the worse ones. I've pushed #18, let's see if having access to `fingertip_last_step` solves your problems.

Reply:
redone; seems to be working fine